src/zeek.bif

```diff
@@ -2721,7 +2721,7 @@ function to_addr%(ip: string%): addr
 		}
 
 	delete [] s;
-	return std::move(ret);
+	return ret;
```

Comment:
Hmm... I wonder if the `ValPtr ret` pattern could be used elsewhere, too, to avoid the noisy `return std::move(r)`: If `ret` was instead `AddrValPtr`, coverity would flag the `std::move()` as necessary, correct?

Nothing to address here, was just taken by surprise why now there's a return in BiFs that doesn't need `std::move()` anymore.